### ISSUE TITLE
<damage>d letters, encoding issue from RIE inscription

### ISSUE THREAD

**Issue by PietroLiuzzo** [BetaMasaheft/Documentation#1748]
Dear @abausi @DariaElagina @DenisNosnitsin1970 @eu-genia @smaugustine and @thea-m and all

we are encoding some ethiopic inscriptions for BM (see manuscript repository, branch 'angelainscriptions'). @angelasalerno , from the University of Bologna, who is working on this for her Erasmus Training in Hamburg on these texts using as reference the EpiDoc guidelines (on which BM was based), has pointed out the following encoding issue.

The editors mark with partial brackets the parts of readable letters which are damaged 'les demi-crochets indiquent les parties abîmées des lettres'. See for example the image below and the transcription from RIE I, 2.

<img width="494" alt="Schermata 2021-05-14 alle 10 17 28" src="https://user-images.githubusercontent.com/508012/118519785-b73a4400-b739-11eb-8c49-c46d4cf83975.png">
<img width="640" alt="Schermata 2021-05-14 alle 10 17 38" src="https://user-images.githubusercontent.com/508012/118519798-b99c9e00-b739-11eb-8305-1a6683168d3b.png">

We think these should be marked up with [`<damage>`](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-damage.html) which however, alone does not provide the information on the place of the damage, which is instead given in the edition. After consultation on the markup list (EpiDoc), where it was suggested not to use `@extent` or add `@place`, but rather to use a namespaced attribute local to our spec I would like to add a custom attribute `@placeOfDamage` to the element (which is already available in our TEI customization)

```xml
<lb n='1'/> ..... mšrq<damage placeOfDamage="upperLeft lowerLeft">h</damage> .......

<lb n='2'/> ..... wʾ<damage placeOfDamage="lowerRight">b</damage> ......
```

This will have a controlled vocabulary of values including 'upperLeft', 'lowerLeft', 'upperRight', 'lowerRight' to be used and will be restricted to use only with single characters. The schema will contain a specification like the following.

```xml
<elementSpec ident="damage" module="transc" mode="change">
                    <constraintSpec scheme="isoschematron" ident="date">
                        <constraint>
                            <sch:rule context="tei:damage[@placeOfDamage]">
                                <sch:report test="string-length(text()) gt 1"
                                    see="https://betamasaheft.eu/Guidelines/?id=inscriptions">The usage of the attribute placeOfDamage is relative to one letter, please replicate the element damage with this attribute for each notated letter. </sch:report>
                            </sch:rule>                        </constraint>
                    </constraintSpec>
                    <attList>
                        <attDef ident="placeOfDamage" 
                            ns="https://betamasaheft.eu" usage="opt" mode="add">
                            <desc>Place of the damage of a letter. This has been added to represent the half brackets of RIE, used in the edition to mark <foreign xml:lang="fr">les parties abîmées des lettres</foreign> (RIÉ I, 13).</desc>
                            <datatype maxOccurs="4">
                                <dataRef key="teidata.enumerated"/>
                            </datatype>
                            <valList type="open">
                                <valItem ident="upperRight"/>
                                <valItem ident="upperLeft"/>
                                <valItem ident="lowerRight"/>
                                <valItem ident="lowerLeft"/>
                            </valList>
                        </attDef>
                    </attList>
                    <exemplum>
                        <egXML xmlns="http://www.tei-c.org/ns/Examples" 
                            xmlns:bm="https://betamasaheft.eu" source="RIE7566dasi">
                        mšrq<damage bm:placeOfDamage="upperLeft lowerLeft">h</damage>    
                    </egXML>
                    </exemplum>
                </elementSpec>
```

The visualization in the app will use corresponding half brackets.  

This may be used for the encoding of manuscripts as well, of course and any other written artefact, but will for sure find examples in the inscriptions, so that the guidelines can be updated accordingly.

Using this attribute will also require the addition of the `xmlns:bm="https://betamasaheft.eu"` in the file.

Please let us know what you think on any of the above.

**Comment by DariaElagina** [comment]
From my point of view, your solution sounds great! I can just imagine that there might be also a need of encoding the completely damaged upper or lower part of a letter, and to distinguish it from the situation when a letter is damaged on the corners (upperLeft upperRight) only, but the central upper part has been preserved. Just as an idea.

**Comment by DariaElagina** [comment]
![Damage_Ep](https://user-images.githubusercontent.com/29276221/119020367-71ca8080-b99e-11eb-9af1-88375cc7cef2.jpg)
I am pretty bad at drawing, but I tried to highlight the difference in the types of damage I am talking about. My experience with inscriptions is very limited, however.

**Comment by eu-genia** [comment]
Sorry am late :) @DariaElagina is there ever such a distinction as you wish done in editions? In RIE I believe they would do upperLeft upperRight for both... @PietroLiuzzo do other epigraphists have the means to mark this up?
Otherwise the suggestion to encode what we have in RIE sounds very good to me.

**Comment by PietroLiuzzo** [comment]
I have enquired with other epigraphic encoders and they do not have this, the suggestion above was in any case coordinated with other epigraphic encoding projects in that way.

**Comment by DariaElagina** [comment]
@eu-genia , no, it was rather a theoretical question. Sorry, for any confusion I have caused :)
